src/cljs/rems/table.cljs

```diff
@@ -18,19 +18,15 @@
 (defn column-class [column-definitions col]
   (get-in column-definitions [col :class] (name col)))
 
-(defn column-class-customized [item column-definitions col]
-  (if (= "draft" (:state item))
-    "text-highlight"
-    (column-class column-definitions col)))
-
 (defn column-sort-value [column-definitions col item]
   ((or (get-in column-definitions [col :sort-value])
        (get-in column-definitions [col :value])) item))
 
-(defn- row [column-definitions columns item]
-  (into [:tr.action]
+(defn- row [{:keys [row-class]} column-definitions columns item]
```

Comment:
why not just a normal argument? would simplify this and the calling code

Reply:
It is a capability to give row options. There might be more in the future. :D

Reply:
hmm ok. please at least document these parameter changes in the `component` docstring

Reply:
Ended up removing the row-class, as it was not used.